test/sol5/kyberMatchingEngine.js

```diff
@@ -274,31 +279,23 @@ contract('KyberMatchingEngine', function(accounts) {
         });
 
         describe("test cases where reserve has never been added", async() => {
-            it("should revert for zero reserve id", async() => {
-                let zeroReserveId = "0x0";
-                await expectRevert(
-                    matchingEngine.addReserve(reserve.address, zeroReserveId, reserve.onChainType, {from: network}),
-                    "reserveId = 0"
-                );
-            });
-    
             it("should revert for NONE reserve type", async() => {
                 await expectRevert(
-                    matchingEngine.addReserve(reserve.address, reserve.reserveId, 0, {from: network}),
+                    matchingEngine.addReserve(reserve.reserveId, 0, {from: network}),
                     "bad reserve type"
                 );
             });
     
             it("should revert for LAST reserve type", async() => {
                 await expectRevert(
-                    matchingEngine.addReserve(reserve.address, reserve.reserveId, 0, {from: network}),
+                    matchingEngine.addReserve(reserve.reserveId, 7, {from: network}),
                     "bad reserve type"
                 );
             });
     
             it("should revert for valid reserve because fee paying data not set", async() => {
                 await expectRevert(
-                    matchingEngine.addReserve(reserve.address, reserve.reserveId, reserve.onChainType, {from: network}),
+                    matchingEngine.addReserve(reserve.reserveId, reserve.onChainType, {from: network}),
```

Comment:
don't understand parameter name onChainType
I mean offChainType is different.

please rename.
maybe reserve type?